package.json

```diff
@@ -24,7 +24,7 @@
   "main": "index.js",
   "scripts": {
     "start": "node example/index.js",
-    "test": "./node_modules/.bin/mocha"
+    "test": "./node_modules/.bin/mocha test/**/*.spec.js test/renderers/**/*.spec.js"
```

Comment:
Yep, I actually did that already. Initially I added the second argument because I had an issue with the glob pattern getting expanded by the shell instead of passing it to mocha as is. So it covered